app/assets/javascripts/dashboard/explorative_annotations_view.js

```diff
@@ -36,6 +36,13 @@ const { Column } = Table;
 const { Search } = Input;
 
 const typeHint: APIAnnotationTypeCompact[] = [];
+const pageLength: number = 1000;
+
+export type TracingMode = {
```

Comment:
I'd prefer a `State` suffix here as well (and in all state variables which refer to this type).